src/cms/templates/_base.html

```diff
@@ -61,16 +61,32 @@
         </div>
     </div>
     <div id="user-info" class="relative px-2 text-gray-800 flex flex-col justify-center cursor-pointer hover:bg-gray-200">
-        <a href="{% url 'user_settings' %}" class="relative block whitespace-nowrap">
-            <i data-feather="user"></i>
-            {{ request.user.profile.full_user_name }}
-            <i data-feather="chevron-down"></i>
-        </a>
+        {% if region %}
+            <a href="{% url 'user_settings' region_slug=region.slug%}" class="relative block whitespace-nowrap">
+                <i data-feather="user"></i>
+                {{ request.user.profile.full_user_name }}
+                <i data-feather="chevron-down"></i>
+            </a>
+        {% else %}
+            <a href="{% url 'user_settings' %}" class="relative block whitespace-nowrap">
+                <i data-feather="user"></i>
+                {{ request.user.profile.full_user_name }}
+                <i data-feather="chevron-down"></i>
+            </a>
+        {% endif %}
```

Comment:
```suggestion
        <a href="{% if region %}{% url 'user_settings' region_slug=region.slug %}{% else %}{% url 'user_settings' %}{% endif %}" class="relative block whitespace-nowrap">
            <i data-feather="user"></i>
            {{ request.user.profile.full_user_name }}
            <i data-feather="chevron-down"></i>
        </a>
```
I think, in this case I'd prefer the slightly more concise syntax. But your solution is also fine.